server/routerlicious/packages/lambdas-driver/src/document-router/documentPartition.ts

```diff
@@ -133,6 +134,14 @@ export class DocumentPartition {
      */
     private markAsCorrupt(message: IQueuedMessage, error: any) {
         this.corrupt = true;
+        this.context.log?.error(
+            `Marking document as corrupted due to error: ${inspect(error)}`,
+            {
+                messageMetaData: {
+                    documentId: this.documentId,
```

Comment:
these are likely PII and cannot be logged

Reply:
TenantId and DocumentId? I didn't think so, do you have information backing that up?
We log that in multiple other places throughout the code. A couple of examples:
https://github.com/microsoft/FluidFramework/blob/cd3c8a8f3419f0dd3f68fba4f73024dfad4a12e8/server/gitrest/src/externalStorageManager.ts#L52
https://github.com/microsoft/FluidFramework/blob/cd3c8a8f3419f0dd3f68fba4f73024dfad4a12e8/server/routerlicious/packages/lambdas/src/deli/lambdaFactory.ts#L86
https://github.com/microsoft/FluidFramework/blob/cd3c8a8f3419f0dd3f68fba4f73024dfad4a12e8/server/routerlicious/packages/lambdas/src/alfred/index.ts#L317

Reply:
If that is PII, we have a lot of work to do to clean it up

Reply:
my guess is that it is PII? is it user specified?

Reply:
in the runtime code we log a doc id, but it is actually a hash of the documents id, as doc id in PII. tenant might be OII if it's already public, but i usually consider anything like that to be PII

Reply:
i just recommend being careful here, as it only gets harder to fix things like this if customer data is logged

Reply:
According to the "Microsoft Enterprise Online Services Data Taxonomy (aka Asset Classification Standard)", "Tenant ID (non-GUID)" is Organization Identifiable Information (OII). On the other hand, "Session IDs" are "End User Pseudonymous Identifiers (EUPI)" as long as it's not data created by the customer (C1 or C2). But I think that documentId might be specified by the application (C1 in this case?)

Do we have any privacy contact?

Reply:
@ahmedbisht for more insight on this.

Reply:
Update here: our team discussed this offline with @ahmedbisht and we will follow-up on this later. This change does not make things worse as we are already logging documentID (which is customer content) in many other places. We are working to make a plan around how to fix that, and we will fix all occurrences at once later. I created this work item to track that task. #6016